app/scripts/controllers/app-state.js

```diff
@@ -100,7 +106,8 @@ export default class AppStateController extends EventEmitter {
         }),
     );
 
-    const { preferences } = preferencesStore.getState();
+    const { preferences } = preferencesController.state;
```

Comment:
Can we use the messenger here, so we avoid passing the controller instance to constructor options?

Reply:
@kanthesha we don't need a new ticket for this it should be done part of the work on this ticket can you instead add a comment or update the description https://github.com/MetaMask/metamask-extension/issues/25916